Local build failing with Property 'value' does not exist on type 'EventTarget'

Local builds are failing for me when running `npm run compile:dev`

macOS/node 6.1.3/npm 3.10.3

```
RROR in ./app/src/ui/create-branch/index.tsx
(80,30): error TS2339: Property 'value' does not exist on type 'EventTarget'.

ERROR in ./app/src/ui/create-branch/index.tsx
(97,41): error TS2339: Property 'value' does not exist on type 'EventTarget'.

ERROR in ./app/src/ui/changes/commit-message.tsx
(29,29): error TS2339: Property 'value' does not exist on type 'EventTarget'.

ERROR in ./app/src/ui/changes/commit-message.tsx
(37,33): error TS2339: Property 'value' does not exist on type 'EventTarget'.

ERROR in ./app/src/ui/branches/index.tsx
(69,31): error TS2339: Property 'value' does not exist on type 'EventTarget'.

ERROR in ./app/src/ui/add-repository/create-repository.tsx
(29,31): error TS2339: Property 'value' does not exist on type 'EventTarget'.

ERROR in ./app/src/ui/add-repository/create-repository.tsx
(34,31): error TS2339: Property 'value' does not exist on type 'EventTarget'.

ERROR in ./app/src/ui/add-repository/add-existing-repository.tsx
(56,31): error TS2339: Property 'value' does not exist on type 'EventTarget'.
```

/cc @shiftkey 👀

--- a/package.json
+++ b/package.json
@@ -78,7 +78,7 @@
     "@types/mocha": "^2.2.29",
     "@types/moment": "^2.11.29",
     "@types/node": "^6.0.31",
-    "@types/react": "^0.14.28",
+    "@types/react": "0.14.28",
     "@types/react-addons-test-utils": "^0.14.14",
     "@types/react-dom": "^0.14.14",
     "@types/whatwg-fetch": "0.0.28"

